applications/GeoMechanicsApplication/tests/cpp_tests/test_utilities.h

```diff
@@ -19,3 +19,21 @@ constexpr auto absolute_tolerance = 1.0e-12;
 constexpr auto relative_tolerance = 1.0e-6;
 
 } // namespace Kratos::Testing::Defaults
+
+namespace Kratos::Testing
+{
+
+template <typename PointContainerType1, typename PointContainerType2>
+void ExpectPointsAreNear(const PointContainerType1& rPoints1,
+                         const PointContainerType2& rPoints2,
+                         double AbsoluteTolerance = Defaults::absolute_tolerance)
+{
+    ASSERT_EQ(rPoints1.size(), rPoints2.size());
+    for (auto i = std::size_t{0}; i < rPoints1.size(); ++i) {
+        KRATOS_EXPECT_NEAR(rPoints1[i].X(), rPoints2[i].X(), AbsoluteTolerance);
+        KRATOS_EXPECT_NEAR(rPoints1[i].Y(), rPoints2[i].Y(), AbsoluteTolerance);
+        KRATOS_EXPECT_NEAR(rPoints1[i].Z(), rPoints2[i].Z(), AbsoluteTolerance);
```

Comment:
I think we're missing some includes here. I'd propose to change this to the gtest default:
```suggestion
        EXPECT_NEAR(rPoints1[i].X(), rPoints2[i].X(), AbsoluteTolerance);
        EXPECT_NEAR(rPoints1[i].Y(), rPoints2[i].Y(), AbsoluteTolerance);
        EXPECT_NEAR(rPoints1[i].Z(), rPoints2[i].Z(), AbsoluteTolerance);
```
And include #include <gtest/gtest.h> in this file (although also the `expect.h` would not be as large as e.g. `testing/testing.h`

Reply:
We have together fixed the problem with the missing include. Thank you.

Also, the Kratos Core macro `KRATOS_EXPECT_NEAR` has been replaced by `EXPECT_NEAR` as per your suggestion. We need, however, define a policy on which macros to use, to avoid inconsistencies in the test code.